VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/DeveloperScreen/OverrideApiScreen.tsx

```diff
@@ -0,0 +1,551 @@
+import React, { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useSelector } from 'react-redux'
+
+import { StackScreenProps } from '@react-navigation/stack'
+
+import { Button } from '@department-of-veterans-affairs/mobile-component-library'
+import { TFunction } from 'i18next'
+import _ from 'lodash'
+
+import { errorOverride, errors } from 'api/types'
+import { Box, FeatureLandingTemplate, SelectorType, TextArea, TextView, VASelector, VATextInput } from 'components'
+import { NAMESPACE } from 'constants/namespaces'
+import { HomeStackParamList } from 'screens/HomeScreen/HomeStackScreens'
+import { RootState } from 'store'
+import { DemoState, updateErrorOverrides } from 'store/slices/demoSlice'
+import { VATheme } from 'styles/theme'
+import { useAppDispatch, useTheme } from 'utils/hooks'
+
+type OverrideAPIScreenProps = StackScreenProps<HomeStackParamList, 'OverrideAPI'>
+
+const IndividualQueryDisplay = (
+  endpoint: string,
+  overrideErrors: errors[],
+  setErrors: React.Dispatch<React.SetStateAction<errors[]>>,
+  clearErrors: boolean,
+  t: TFunction,
```

Comment:
We don't need to pass the translation function since this is a component. It can be accessed directly with the `useTranslation` hook, which would make the usage of this component cleaner

Reply:
I removed translation and theme being passed in and I am calling it in the component.